synthetify/index.js

```diff
@@ -0,0 +1,18 @@
+const retry = require('async-retry')
+const axios = require("axios");
+
+async function fetch() {
+    const response = (
+      await retry(async () => await axios.get("https://api.synthetify.io/stats/mainnet"))
+    ).data;
+    const index = response.length - 1
+    return response[index].collateralAll + response[index].synthetic
```

Comment:
Isn't this adding the collateral and the debt together as TVL? This seems different that the methodology explanation, which outlines that only collateral is counted